docs/en/02-how-to-run/convert_model.md

```diff
@@ -65,7 +65,7 @@ python ./tools/deploy.py \
 
 ## How to evaluate the exported models
 
-You can try to evaluate model, referring to [how_to_evaluate_a_model](./how_to_evaluate_a_model.md).
+You can try to evaluate model, referring to [how_to_evaluate_a_model](./profile_model.md).
```

Comment:
`How to evaluate` originally meant test the model on a dataset to get all metrics, which would use `tools/test.py`.